spec/invoice_repository_spec.rb

```diff
@@ -15,7 +15,7 @@
 
   describe '#all' do
     it 'has an array of all invoices' do
-      allow_any_instance_of(InvoiceRepository).to receive(:create_invoices).and_return(MockData.invoices_as_hashes)
+      allow_any_instance_of(InvoiceRepository).to receive(:create_invoices).and_return(InvoiceMocks.invoices_as_hashes)
```

Comment:
@redferret we should be breaking up these lines like this, I think:

```rb
mock_hashes = InvoiceMocks.invoices_as_hashes
allow_any_instance_of(InvoiceRepository).to receive(:create_invoices).and_return(mock_hashes)
```

Reply:
With a variable, I agree. I won't do that for this refactor since it is just changing names everywhere atm but let's definitely put a refactor job out for this file

Reply:
Created an issue for this: https://github.com/redferret/black_thursday/issues/86